qa/SidechainTestFramework/scutil.py

```diff
@@ -507,22 +538,23 @@ def bootstrap_sidechain_nodes(dirname, network=SCNetworkConfiguration):
 Parameters:
  - sc_creation_info: an instance of SCCreationInfo (see sc_boostrap_info.py)
  
- Output: a map of:
+ Output:
   - an instance of SCBootstrapInfo (see sc_boostrap_info.py)
 """
 def create_sidechain(sc_creation_info):
     account_secrets = generate_secrets(sc_creation_info.sidechain_id, 1)
     genesis_secret = account_secrets[0]["secret"]
     genesis_public_key = account_secrets[0]["publicKey"]
     sidechain_id = sc_creation_info.sidechain_id
-    genesis_info = get_genesis_info(sidechain_id,
+    genesis_info = initialize_new_sidechain_in_mainchain(sidechain_id,
                                     sc_creation_info.mc_node,
                                     sc_creation_info.withdrawal_epoch_length,
                                     account_secrets,
```

Comment:
just one public key instead of array of `account_secrets`?

Reply:
Replaced with one public key and one amount